features/src/test/scala/com/salesforce/op/stages/base/binary/BinaryTransformerTest.scala

```diff
@@ -42,10 +42,14 @@ class BinaryTransformerTest extends OpTransformerSpec[Real, BinaryTransformer[Re
   val sample = Seq(Real(1.0) -> RealNN(0.0), Real(2.0) -> RealNN(2.0), Real.empty -> RealNN(1.0))
   val (inputData, f1, f2) = TestFeatureBuilder(sample)
 
-  val transformer = new BinaryLambdaTransformer[Real, RealNN, Real](operationName = "bmi",
-    transformFn = (i1, i2) => new Real(for { v1 <- i1.value; v2 <- i2.value } yield v1 / (v2 * v2))
+  val transformer = new BinaryLambdaTransformer[Real, RealNN, Real](
```

Comment:
General question for our new transformers. I thought we could still use lambdas, they just won't serialize when we save the model so the only way to load them is if you also have the workflow. Or is that not the case?

If we still can use lambdas in our transformers as long as we don't try to save/load models, then are you rewriting all the tests to follow this pattern just so people don't see a version that will suddenly break when they try and load things?